src/rules/alias.ts

```diff
@@ -112,7 +112,12 @@ const rule = {
 				);
 				if (!isPathInImport) return;
 
-				const filename = context.filename;
+				let filename = context.filename;
+				if (typeof filename === 'undefined') {
```

Comment:
Currently this code just hides a problem for exists test cases.

Let's add tests that reproduce original problem from issue, and make sure all other tests will be run for that case too.